From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ricardo Pardini <ricardo@pardini.net>
Date: Mon, 9 Jan 2023 02:18:23 +0100
Subject: BananaPi M5: 270 clock phase, via amlogic,mmc-phase

BananaPi M5: 270 clock phase, via amlogic,mmc-phase
- rpardini: adapted for 6.4
---
 arch/arm64/boot/dts/amlogic/meson-sm1-bananapi-m5.dts | 12 ++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-sm1-bananapi-m5.dts b/arch/arm64/boot/dts/amlogic/meson-sm1-bananapi-m5.dts
index f045bf851638..0c261b12ed8b 100644
--- a/arch/arm64/boot/dts/amlogic/meson-sm1-bananapi-m5.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-sm1-bananapi-m5.dts
@@ -9,6 +9,7 @@
 #include "meson-sm1-bananapi.dtsi"
 #include <dt-bindings/sound/meson-g12a-toacodec.h>
 #include <dt-bindings/sound/meson-g12a-tohdmitx.h>
+#include <dt-bindings/mmc/meson-gx-mmc.h>
 
 / {
 	compatible = "bananapi,bpi-m5", "amlogic,sm1";
@@ -219,3 +220,14 @@ &toddr_b {
 &toddr_c {
 	status = "okay";
 };
+
+/* SD card */
+&sd_emmc_b {
+	amlogic,mmc-phase = <CLK_PHASE_270 CLK_PHASE_0 CLK_PHASE_0>;
+};
+
+/* eMMC */
+&sd_emmc_c {
+	amlogic,mmc-phase = <CLK_PHASE_270 CLK_PHASE_0 CLK_PHASE_0>;
+};
+
-- 
Armbian

